hs/Database/Tables.hs

```diff
@@ -140,6 +141,13 @@ FacebookTest
 
 -- ** TODO: Remove these extra types and class instances
 
+-- | JSON SVG data
+data SvgJSON =
+    SvgJSON { texts :: [Text],
```

Comment:
See `TODO` above. Is it possible to modify the database schema instead of creating the extra type?